README.md

```diff
@@ -178,3 +180,37 @@ The generated `image` name follows the pattern
 For example: `epimorphics/myapp/production`
 
 The `aws` information is extracted from the specification by this action just to avoid later workflow steps having to do a repeat parse. The `aws.region` is optional and defaults to `eu-west-1`.
+
+### Version 2
+
+In version 2 of the `deployment.yaml` fiel the deployement struction is turned inside-out. 
+If the version is not specified version 1 (above) is assumed.
+
+If the name field is present in the deployments.yaml this is returned as `image`.
```

Comment:
Possibly worth explicitly noting here that this is different from the value that version 1 would have returned (no `/environment`).